src/validator.ts

```diff
@@ -1067,11 +1067,12 @@ function recursiveDecent(ref: any){
             // Check if an Intrinsic function is allowed here
             let inResourceProperty = (placeInTemplate[0] == "Resources" || placeInTemplate[2] == "Properties");
             let inResourceMetadata = (placeInTemplate[0] == "Resources" || placeInTemplate[2] == "Metadata");
+            let inGlobalEnvironment = (placeInTemplate[0] == "Globals" && placeInTemplate[2] == "Environment");
```

Comment:
Again, my IDE has made some none requested cleaning, sorry about that. Changes appear here.

Reply:
Can you clean this up before committing? It makes retrospective diff viewing unpleasant.